telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -43,17 +44,60 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        chat_id (:SCT[:obj:`int`], optional): Filters requests to allow only
```

Comment:
Ah, sadly using our shortcut does not work with sphinx, sorry for misleading you there.
``` `:obj:`int` | Collection[:obj:`int`], optional``` was right in the written type hint.

Reply:
there are type aliases in sphinx which would work, but we'd have to make it public first, which isn't sooo